modules/monitoring-example-alertmanager-webhook-authentication-settings.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: REFERENCE
+[id="example-alertmanager-webhook-authentication-settings_{context}"]
+= Example Alertmanager webhook authentication settings
+
+These examples show settings that configure Alertmanager in the `openshift-monitoring` namespace to authenticate with a webhook endpoint by using an API token that is configured in a `Secret` object.
+
+The following sample YAML shows a `Secret` object with authentication credentials to be referenced in the Alertmanager config map settings for the Cluster Monitoring Operator (CMO):
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+data: 
+  api_token: m8VjfmV2Cg== <1>
+metadata: 
+  name: alertmanager-credentials <2>
+  namespace: openshift-monitoring
+----
+<1> Replace this value with the API token used to authenticate to your webhook endpoint.
+<2> Reference this name in the `secrets` field in the CMO config map settings. 
```

Comment:
in this secret, <2> is only the secret name, I think change to below is better:
<2>  `secrets` name that to be  referenced  in the CMO config map settings.